test/conformance/gatewayapi/gateway_conformance_test.go

```diff
@@ -79,15 +84,38 @@ func TestGatewayConformance(t *testing.T) {
 			// implemented plus complications with the test setup itself.
 			// See: https://github.com/projectcontour/contour/issues/5922
 			tests.GatewayStaticAddresses.ShortName,
+
+			// Skip this test as it uses a Gateway with a name that is too long,
+			// adding the name in a label value prevents resources to be
+			// created.
+			tests.HTTPRouteInvalidParentRefSectionNameNotMatchingPort.ShortName,
```

Comment:
🤮 would be nice to change this upstream if possible, have to imagine other folks will run into issues too

Reply:
(vomit face was for the upstream test using too long a name, not you disabling it!)

Reply:
ha yeah i figured, and agreed tbh

I think there was a discussion on this upstream i'll have to find it again